### ISSUE TITLE
Scattering on a large conducting object

### ISSUE THREAD

**Issue by hdierckx** [HomerReid/scuff-em#99]
Hi Homer,

We want to describe scattering off a large homogeneous metal object (few cm) at low frequency (skin depth= few microns). How can we go beyond the PEC-approximation with scuff-scatter?
Should we e.g. use a complex permittivity or can we impose Leontovitch boundary conditions? 

Thank you for your help.

**Comment by HomerReid** [comment]
Yes, for something like this you would probably want to use a surface impedance. This is already implemented in SCUFF-EM, but the implementation is not particularly well tested because I haven't had a good application for it. If you have such an application, I would be happy to use it as an opportunity to polish the implementation and documentation. Is there a simple toy version of your problem, for which you already know the correct answers, that we could use to test the SCUFF calculations?
